prow/cmd/mkbuild-cluster/main.go

```diff
@@ -238,7 +238,7 @@ func do(o options) error {
 		if err != nil {
 			return err
 		}
-		if _, err = c.ListPods("k8s-app=kube-dns"); err != nil {
+		if _, err = c.ListProwJobs("k8s-app=kube-dns"); err != nil {
```

Comment:
(I don't think this will work?)

Reply:
If build clusters don't have ProwJobs registered?

Reply:
Yeesh

Reply:
oh.. I guess this meant to check if there is a kubedns pod exist?

Reply:
Very tempted to remove this check... @fejta ?

Reply:
actually - just use a pod client here instead?